Katana/Store/ActionLinker.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ActionLinker.swift
+//  Katana
+//
+//  Created by Riccardo Cipolleschi on 11/01/2017.
+//  Copyright © 2017 Bending Spoons. All rights reserved.
+//
+
+import Foundation
+
+/**
+ The engine that dispatches all the actions that depends on a triggered source action.
+ This is invoked every time an action is dispatched. 
+ The ActionLinker is responsible to check if there are actions that must be issued after a source actionis dispatched.
+ For each of this actions, it checks if the conditions are met: if yes, they are dispatched.
+ */
+struct ActionLinker {
```

Comment:
I don't agree. 
A user should not know that there is an ActionLinker.
The store init it, you just have to pass the links